src/Compilers/CSharp/Portable/Symbols/NamespaceSymbol.cs

```diff
@@ -294,6 +275,11 @@ internal NamespaceSymbol GetNestedNamespace(string name)
             return null;
         }
 
+        public abstract ImmutableArray<Symbol> GetMembers(ReadOnlyMemory<char> name);
```

Comment:
>public abstract ImmutableArray<Symbol> GetMembers(ReadOnlyMemory<char> name); [](http://example.com/codeflow?start=8&length=77)

It looks like this should be placed in the base class, next to ```GetMembers(string name)``` definition

Reply:
i would prefer to *not* do that.  we only need GetMembers(ROM) support for GetTypeByMetadataName for namespaces.  Moving this up to NamespaceOrTypeSymbol would mean updating a ton of types to now support ROM lookup of members, even though that's not a facility we're needing yet, and we don't see it in traces as an issue.

Reply:
> i would prefer to not do that.  we only need GetMembers(ROM) support for GetTypeByMetadataName for namespaces.  

Ok